wallet/lib/utils/favorites_change_notifier.dart

```diff
@@ -5,22 +5,20 @@ import '../model/nft.dart';
 import '../services/repository/repository.dart';
 import 'enums.dart';
 
-class FavoritesChangeNotifier extends ChangeNotifier{
-
+class FavoritesChangeNotifier extends ChangeNotifier {
   final Repository repository;
   FavoritesChangeNotifier({required this.repository});
 
   List<NFT> favorites = [];
 
-  Future<void> onInit() async {
-
-  }
+  Future<void> onInit() async {}
```

Comment:
What is the use of this function?

Reply:
It will initialize notifier and other resources...I have written it's implementation it's under review will be merged today  inshallah.

Reply:
@Kamran-Khan-Gene is working on it